app/forms/journeys/additional_payments_for_teaching/poor_performance_form.rb

```diff
@@ -10,7 +11,14 @@ class PoorPerformanceForm < Form
       def save
         return false unless valid?
 
-        update!(eligibility_attributes: attributes)
+        update!(eligibility_attributes: attributes.except("journey_session_id"))
+        journey_session.update(answers: {eligibility_attributes: attributes.except("journey_session_id")})
```

Comment:
I don't think we want to nest these answers under eligibility_attributes as the journey session is just a bag of answers and doesn't need to know about eligibility.

```suggestion
        journey_session.update(answers: attributes.except("journey_session_id"))
```

Reply:
I think we'll want a method like this
```
    def update_answers!(args)
      answers.merge!(args)
      save!
    end
```
on `Journeys::Session`